### PR TITLE
Add settings for image size and frequency

### PR DESCRIPTION
**Pull request by AhnafMuttaki**
Added admin settings for choosing image size and image capture frequency.

### CODE REVIEW

File: version.php [hunk 1]
```diff
@@ -26,6 +26,6 @@
 
 $plugin->component = 'quizaccess_proctoring';
 $plugin->release = '1.1.0';
```
Comment (AnowarCST) [inline version.php:28]: Make it 1.1.1

File: rule.php [hunk 2]
```diff
@@ -224,6 +224,32 @@ public function setup_attempt_page($page) {
             $record->status = $attempt;
             $record->timemodified = time();
             $record->id = $DB->insert_record('quizaccess_proctoring_logs', $record, true);
+
+            //////// Get Image Frequency and Image Width ////////
+            $imagefrequencysql = "SELECT * FROM {config_plugins} WHERE plugin = 'quizaccess_proctoring' AND name = 'autoreconfigurefrequency'";
+            $frequencydata = $DB->get_records_sql($imagefrequencysql);
+
+            $frequency = 3*1000;
```
Comment (AnowarCST) [inline rule.php:232]: 3 seconds or 30 seconds?

**Review by AnowarCST — CHANGES_REQUESTED** [review]

File: rule.php (hunk 2 above)
Comment (AhnafMuttaki) [reply rule.php:232]: 3 seconds

File: version.php (hunk 1 above)
Comment (AhnafMuttaki) [reply version.php]: made the release version 1.1.1 as requested.

**Review by AnowarCST — APPROVED** [review]

**Pull request merged**